soh/soh/Enhancements/mods.cpp

```diff
@@ -687,6 +689,47 @@ void RegisterMirrorModeHandler() {
     });
 }
 
+void RegisterPatchNoMSHandler() {
+        GameInteractor::Instance->RegisterGameHook<GameInteractor::OnGameFrameUpdate>([]() { 
```

Comment:
Would it be possible to patch these less often e.g on scene init/file load? or does it need to be on update

Reply:
If you do it that way, when you get MS it doesn't update, so you would still not see it until you did a scene change

Reply:
there is no way to call the unpatching when you get the master sword?

Reply:
I mean, I'm sure there is. I just don't know it. I still barely understand hooks in general

Reply:
You should be able to register an `OnGetItem` hook and check for ITEM_SWORD_MASTER to get that trigger.

Reply:
This works for the unpatching, but then doesn't work for the original patching out the MS.